migrations/Version202203221158521465_qtiItemPci.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+declare(strict_types=1);
+
+namespace oat\qtiItemPci\migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use oat\tao\scripts\tools\migrations\AbstractMigration;
+use Doctrine\Migrations\Exception\IrreversibleMigration;
+use oat\qtiItemPci\model\IMSPciModel;
+use oat\qtiItemPci\scripts\install\RegisterPciMathEntry;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version202203221158521465_qtiItemPci extends AbstractMigration
+{
+
+    public function getDescription(): string
+    {
+        return 'Register new major version of Math Entry PCI with breaking change: single cardinality';
+    }
+
+    public function up(Schema $schema): void
+        {
+            $registry = (new IMSPciModel())->getRegistry();
+            if ($registry->has('mathEntryInteraction')) {
+                /** @noinspection PhpUnhandledExceptionInspection */
+                $registry->removeAllVersions('mathEntryInteraction');
+            }
```

Comment:
issue: we need to keep the previous versions of the PCIs, because if a test is published with a previous major version of the PCI, it will fail.
So you just need to remove those lines.